### PR TITLE
ADD: intersections

### PR DESCRIPTION
**Pull request by jagreene**
image

### CODE REVIEW

**Comment by sihrc** [comment]
- [ ] change back to apiv2

File: test/integration/images.js [hunk 1]
```diff
@@ -261,9 +260,9 @@ describe('Image', function() {
     });
   });
 
-  describe('batchPredictImage', function() {
+  describe('batchanalyzeImage', function() {
```
Comment (madisonmay) [inline test/integration/images.js]: `batchAnalyzeImage` instead, I think

File: test/integration/text.js [hunk 2]
```diff
@@ -350,17 +353,16 @@ describe('BatchText', function() {
     });
   });
 
-  describe('batchKeywords', function() {
-    it('should get the right response format', function(done) {
+  describe('batchanalyzeText', function() {
```
Comment (madisonmay) [inline test/integration/text.js]: `batchAnalyzeText`

File: test/integration/text.js [hunk 3]
```diff
@@ -369,4 +371,54 @@ describe('BatchText', function() {
...
+      })
+      .catch(function(err){
+        done(err);
+        return;
+      });
+    });
+
+    it('should get the right response format in historic mode', function(done) {
+        this.timeout(5000);
+      indico.textTags(examples)
```
Comment (madisonmay) [inline test/integration/text.js]: Maybe call analyzeText here instead?

Comment (madisonmay) [inline test/integration/text.js]: For one less level of nesting.

Comment (sihrc) [inline test/integration/text.js]: probably a good idea.

**Comment by madisonmay** [comment]
Rebased, good to merge on tests pass.

**Pull request merged**